HelpSource/Classes/Function.schelp

```diff
@@ -434,19 +437,23 @@ code::
 ::
 
 
+
 argument::duration
 The duration of the function to plot in seconds. The default is 0.01.
 argument::server
-The Server on which to calculate the plot. This must be running on your local machine, but does not need to be the internal server. If nil the default server will be used.
+The Server on which to calculate the plot. This must be running on your local machine, but does not need to be the internal server. If nil the default server will be used. This argument can also be a group or synth, it then is used as a target (see link::Reference/asTarget::):
```

Comment:
This will make the current PR an API change. Do we want this? We could also do this change later, and have this one in 3.9.

Reply:
I vote for change to argument: target
1. it expresses the power and flexibility of the method very clearly
2. Function.plot is a utility for checking what the signal resulting from a short synthesis snippet looks like; my guess is that in >99% of the cases it will run silently on the default server, and in >99% of those uses no one specifies a server by keyword. 
Building any larger projects by relying on code like
```~myplotter = { ... }.plot(server: ~mySpecialServer);```
seems really unlikely, and would IMHO not be advisable style.
2c adc